### ISSUE TITLE
temperature module lacks icon array ability

### ISSUE THREAD

**Issue by 7m0E7TRWiyc7eJlETAgDR-rp6HLJf2yvSnJLuQL** [Alexays/Waybar#312]
The wiki mentions the possibility of a "format-icons" array option, but the actual code in temperature.cpp lacks this ability (including format options for only temperatureC and temperatureF).  I edited and recompiled with an ugly hack that allows it to work:
```
--       format, fmt::arg("temperatureC", temperature_c), fmt::arg("temperatureF", temperature_f))));
++       format, fmt::arg("temperatureC", temperature_c), fmt::arg("temperatureF", temperature_f), fmt::arg("icon", getIcon(temperature_c - 30))));
```
But I am sure someone in the project could make it much nicer.

**Comment by 7m0E7TRWiyc7eJlETAgDR-rp6HLJf2yvSnJLuQL** [comment]
`--- waybar/trunk/src/Waybar-0.6.2/src/modules/temperature.cpp   2019-05-10 06:40:45.000000000 -0500
+++ waybar.new/trunk/src/Waybar-0.6.2/src/modules/temperature.cpp       2019-05-12 19:17:05.168991020 -0500
@@ -36,7 +36,7 @@
     label_.get_style_context()->remove_class("critical");
   }
   label_.set_markup(fmt::format(
-      format, fmt::arg("temperatureC", temperature_c), fmt::arg("temperatureF", temperature_f)));
+      format, fmt::arg("temperatureC", temperature_c), fmt::arg("temperatureF", temperature_f), fmt::arg("icon", getIcon((temperature_c - 30) / 0.7))));
 }
 
 std::tuple<uint16_t, uint16_t> waybar::modules::Temperature::getTemperature() {`

Here is an updated diff.  Of course, I have manually entered the necessary parameters (idle temp of 30C and a critical of 100C).  I am sure this can be done, better, but I lack the knowledge of C++ to pull it off myself.

**Comment by Alexays** [comment]
Fixed with fcb23d3104d2f36edecdf0866d202fb2110c64f3
PS: Define critical-threshold for better accuracy
